core/variant/variant_call.cpp

```diff
@@ -1814,6 +1814,8 @@ static void _register_variant_builtin_methods() {
 	bind_static_method(Quaternion, from_euler, sarray("euler"), varray());
 	bind_method(Quaternion, get_axis, sarray(), varray());
 	bind_method(Quaternion, get_angle, sarray(), varray());
+	bind_method(Quaternion, xform, sarray("v"), varray());
+	bind_method(Quaternion, xform_inv, sarray("v"), varray());
```

Comment:
I believe we already expose this using the `*` operator. All other types that had `xform` and `xform_inv` methods in 3.x unexposed them in favor of the `*` operator in 4.0.

https://github.com/godotengine/godot/blob/234f8e94ac15c3e4e72d00c1e1cbec0ca5efe95e/core/variant/variant_op.cpp#L363-L364

Reply:
At least Transform3D still has xform, so I think it could exist as an alias.

Reply:
Are you sure? I couldn't find it in in `variant_call.cpp`.

Reply:
Oh , I mistook it for the interpolate_with argument, hmmm, well, I'll remove it.